script.js

```diff
@@ -30,3 +30,38 @@ btnMenuItem2.onclick = function show() {
 btnMenuItem3.onclick = function show() {
   nav.classList.remove('display');
 };
+const email = document.getElementById('email');
+const emailError = document.querySelector('.btnSubmit + small.error');
+const emailRegExp = /^[a-z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-z0-9-]+(?:\.[a-z0-9-]+)*$/;
+
+email.addEventListener('input', (event) => {
+  if (!emailRegExp.test(email.value)) {
+    emailError.innerHTML = 'Email must be in lowercase';
+    document.querySelector('.btnSubmit').type = 'button';
+  } else {
+    emailError.innerHTML = '';
+    document.querySelector('.btnSubmit').type = 'submit';
+  }
+  event.preventDefault();
+});
+
+const btnSubmit = document.querySelector('.btnSubmit');
+btnSubmit.onclick = function storeData() {
+  const name = document.querySelector('#name').value;
+  const comment = document.querySelector('#comment').value;
+  const email = document.querySelector('#email').value;
+  const userInfo = {
+    name,
+    email,
+    comment,
+  };
+  const string = JSON.stringify(userInfo);
+  localStorage.setItem('userInfo', string);
+};
```

Comment:
Well done on using `localStorage` 💯 but for now the data is only getting stored in `localStorage` once the `form` is submitted 🚀 yet the requirement states that after filling the form, and not submitting it, the entered values should be saved in the browser. 🕸️ So even if the user started typing and the browser closed for some reason, the user should still be able to retrieve the data. I suggest looking into [onchange event](https://www.w3schools.com/jsref/event_onchange.asp) 🔗